controllers/usersignup/usersignup_controller.go

```diff
@@ -771,7 +771,7 @@ func (r *Reconciler) sendDeactivatedNotification(logger logr.Logger, config tool
 		}
 
 		notification, err := notify.NewNotificationBuilder(r.Client, userSignup.Namespace).
-			WithTemplate(notificationtemplates.UserDeactivated.Name).
+			WithTemplate(notificationtemplates.SandboxUserDeactivated.Name).
```

Comment:
And here.

Reply:
replaced with constant [3df1eab](https://github.com/codeready-toolchain/host-operator/pull/754/commits/3df1eab38ec6e59824567e8d6c9d39bd6c43a3ff)
Again, this is just template name, actual fetching based on notification env is done while sending.